src/js/view/view.js

```diff
@@ -102,6 +103,41 @@ define([
             document.msExitFullscreen;
         _elementSupportsFullscreen = _requestFullscreen && _exitFullscreen;
 
+        function _applyStyles() {
+            var pollTarget = utils.createElement(
+                '<span class="jw-font-load-detect jw-reset">&#xe614&#xe614&#xe614</span>'
+            );
+            _playerElement.appendChild(pollTarget);
+            utils.css(pollTarget, {
```

Comment:
This adds a stylesheet to the DOM. `utils.style` adds styles to the element's styles attribute, inline.